packages/language-server/src/plugins/typescript/service.ts

```diff
@@ -294,22 +280,25 @@ async function createLanguageService(
         getNewLine: () => tsSystem.newLine,
         resolveTypeReferenceDirectiveReferences:
             svelteModuleLoader.resolveTypeReferenceDirectiveReferences,
-        hasInvalidatedResolutions: svelteModuleLoader.mightHaveInvalidatedResolutions
+        hasInvalidatedResolutions: svelteModuleLoader.mightHaveInvalidatedResolutions,
+        getModuleResolutionCache: svelteModuleLoader.getModuleResolutionCache
     };
 
     const documentRegistry = getOrCreateDocumentRegistry(
         host.getCurrentDirectory(),
         tsSystem.useCaseSensitiveFileNames
     );
 
-    const languageService = ts.createLanguageService(host, documentRegistry);
     const transformationConfig: SvelteSnapshotOptions = {
         parse: svelteCompiler?.parse,
         version: svelteCompiler?.VERSION,
         transformOnTemplateError: docContext.transformOnTemplateError,
         typingsNamespace: raw?.svelteOptions?.namespace || 'svelteHTML'
     };
 
+    const project = initLsCacheProject();
+    const languageService = ts.createLanguageService(host, documentRegistry);
```

Comment:
`initLsCacheProject` also creates a language service, why are we creating another one?